plugin/internal/remotecall/StringMapFlag.go

```diff
@@ -0,0 +1,33 @@
+package remotecall
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+type StringMapFlag struct {
+	cli.Generic
+	Value map[string]string
```

Comment:
I think map should be protected with mutex

Reply:
can you tell why - it is a init time flag for the service